src/app/clusters/icd-management-server/icd-management-server.cpp

```diff
@@ -46,9 +50,10 @@ class IcdManagementAttributeAccess : public app::AttributeAccessInterface
     IcdManagementAttributeAccess() : AttributeAccessInterface(MakeOptional(kRootEndpointId), IcdManagement::Id) {}
 
     CHIP_ERROR Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+    CHIP_ERROR Write(const ConcreteDataAttributePath & aPath, AttributeValueDecoder & aDecoder) override;
 
 private:
-    CHIP_ERROR ReadRegisteredClients(EndpointId endpoint, AttributeValueEncoder & encoder);
+    CHIP_ERROR ReadExpectedClients(EndpointId endpoint, AttributeValueEncoder & encoder);
```

Comment:
Final name for the attribute is `RegisteredClient`. We shouldn't be renaming

Reply:
```suggestion
    CHIP_ERROR ReadRegisteredClients(EndpointId endpoint, AttributeValueEncoder & encoder);
```

Reply:
My bad. This change is unintentional, I started working on this file before the rename, and I messed-up the rebase. Renamed.